src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/FinancialTransactionUpdateControllerTest.java

```diff
@@ -0,0 +1,151 @@
+package pl.byczazagroda.trackexpensesappbackend.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionDTO;
+import pl.byczazagroda.trackexpensesappbackend.dto.UpdateFinancialTransactionDTO;
+import pl.byczazagroda.trackexpensesappbackend.dto.error.ErrorResponse;
+import pl.byczazagroda.trackexpensesappbackend.mapper.FinancialTransactionModelMapper;
+import pl.byczazagroda.trackexpensesappbackend.service.FinancialTransactionService;
+import pl.byczazagroda.trackexpensesappbackend.service.FinancialTransactionServiceImpl;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.util.Objects;
+
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.patch;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static pl.byczazagroda.trackexpensesappbackend.model.FinancialTransactionType.EXPENSE;
+
+@WebMvcTest(controllers = FinancialTransactionController.class,
+        excludeFilters = @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = FinancialTransactionServiceImpl.class),
+        includeFilters = @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = {FinancialTransactionModelMapper.class, ErrorResponse.class}))
+@ActiveProfiles("test")
+public class FinancialTransactionUpdateControllerTest {
+
+    public static final long ID_1L = 1L;
+
+    public static final BigDecimal AMOUNT_POSITIVE = new BigDecimal(1000);
+
+    public static final BigDecimal AMOUNT_NEGATIVE = new BigDecimal(-1000);
+
+    public static final BigDecimal BAD_AMOUNT_AFTER_DECIMAL_POINT = new BigDecimal(990.1234);
```

Comment:
When other "AMMOUNT" constants got name convention already... maybe it's good to name another constant with that convention as well?
`BAD_AMOUNT_AFTER_DECIMAL_POINT` -> `AMOUNT_WITH_WRONG_DECIMAL_PART`

Reply:
Done